docs/source/installation_non_conda.rst

```diff
@@ -13,11 +13,11 @@ Step 1: Install some build dependencies
 .. code-block:: bash
 
     $ apt-get update
-    $ apt-get install -y libgoogle-glog-dev curl build-essential cmake git automake libgmp3-dev libtool ssh libyaml-dev realpath wget valgrind software-properties-common unzip
+    $ apt-get install -y libgoogle-glog-dev curl build-essential cmake git automake libgmp3-dev libtool ssh libyaml-dev realpath wget valgrind software-properties-common unzip libz-dev
 
 .. note::
 
-    You might additionally need to install :code:`cmake3` and :code:`libz-dev`, if these packages are not found by apt-get, you can proceed otherwise install them by running :code:`apt-get install libz-dev`
+    You might additionally need to install :code:`cmake3`, if this package is not found by apt-get, you can proceed.
```

Comment:
When you _might_ need it?  Some ubuntu versions have it while others don't?

Reply:
yes